.github/workflows/publish_maven.yml

```
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+permissions: read-all
```

nit: Probably it would be ok with just
```suggestion
permissions:
  contents: read
```